### PR TITLE
ci: run tests and lint

### PR DESCRIPTION
**Pull request by dianwen**

### CODE REVIEW

File: versions/v1/template/server.ts [hunk 1]
```diff
@@ -1,91 +1,115 @@
...
+  HelpersMap,
+  ResolversMap,
+} from "./types";
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 let resolvers = {};
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 let typeDefs = "";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 let helpers = {};
```
Comment (dianwen) [inline versions/v1/template/server.ts:31]: Why are these set later but not used? Is this right?

File: versions/v1/template/server.ts [hunk 2]
```diff
@@ -1,120 +1,160 @@
...
-    });
+  await apolloServer.start();
+  return expressMiddleware(apolloServer, {
+    context: async () => apolloServerContext,
+  });
 };
 
-export const startStandaloneServer = async (config: CreateServerConfig, typeDefs?: string = typeDefs, resolvers?: ResolversMap = resolvers, helpers?: HelpersMap = helpers): Promise<string> => {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const startStandaloneServer = async (
```
Comment (dianwen) [inline versions/v1/template/server.ts:147]: Not sure why this isn't used but I disabled the linting.

File: versions/v1/template/server.ts (hunk 1 above)
Comment (androbwebb) [reply versions/v1/template/server.ts:31]: They're used as default params.

https://github.com/satsuma-xyz/satsuma-querier-codegen/pull/4/files#diff-1665f4aa86c7f2c5bb63c31ea409c9167edcdea31c14db5cdbd3dfcdf1fa4e83R81-R82

https://github.com/satsuma-xyz/satsuma-querier-codegen/pull/4/files#diff-1665f4aa86c7f2c5bb63c31ea409c9167edcdea31c14db5cdbd3dfcdf1fa4e83R103-R104

**Review by androbwebb — APPROVED** [review]
LGTM. That thing about unused vars seems weird... they're definitely used.

File: versions/v1/template/server.ts (hunk 2 above)
Comment (androbwebb) [reply versions/v1/template/server.ts:147]: It's used by the CLI & Querier I believe. Just not used within this package

File: versions/v1/template/server.ts (hunk 1 above)
Comment (dianwen) [reply versions/v1/template/server.ts:31]: Cool, i'll leave the lint disabling here and let you remove in your PR.

File: versions/v1/template/server.ts (hunk 2 above)
Comment (dianwen) [reply versions/v1/template/server.ts:147]: I suspect it's a linter bug. I aliased the apollo import with the same name and the lint error went away. `import { startStandaloneServer as startStandaloneServerApollo } from "@apollo/server/standalone";`

**Pull request merged**